### DISCUSSION TITLE
tuiTextfieldMaxLength v.3

### DISCUSSION THREAD

**Discussion by expdts** [taiga-family/taiga-ui#3153] [Q&A] [answered]
In v. 2, there was attribute tuiTextfieldMaxLength. In v. 3 he doesn't work. Please tell me which attribute should be used in v. 3 in place tuiTextfieldMaxLength?

**Comment by nsbarsukov** [comment, marked as answer]
@expdts 

Hi!

You can use native attribute:

```html
<tui-input formControlName="testValue">
    Label
    <input
        tuiTextfield
        [attr.maxLength]="30"
    />
</tui-input>
```

In 3.0 we drop `tuiTextfield***`-controllers which can be replaced with native attributes.
